src/config.h

```diff
@@ -0,0 +1,75 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#ifdef SENSORS_LIGHT_PIN
+  #define HAVE_LIGHT
+#endif
+
+#ifdef SENSORS_CO2_PIN
+  #define HAVE_CO2
+#endif
+
+#ifdef SENSORS_EC_PIN
+  #define HAVE_EC
+#endif
+
+#ifdef SENSORS_PH_PIN
+  #define HAVE_PH
+#endif
+
+#ifdef SENSORS_MOISTURE_PIN
+  #define HAVE_MOISTURE
+#endif
+
+#ifdef SENSORS_DHT22_PIN
+  #define HAVE_TEMP_HUMIDITY
+
+  // ensure only digital pins configured
+  #ifdef ARDUINO
+    #if SENSORS_DHT22_PIN < 0 || SENSORS_DHT22_PIN > 13
+      #error "Pin configured for DHT22 must a digital one.")
```

Comment:
Can we change this to: "Pin configured for DHT22 must be digital."

Reply:
Yes, reviews are fine. It's why I am doing PRs.
Fixed the message and the one below which was a wrong copy.